web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -1005,6 +1033,188 @@ function AdminRule({
+        grid-template-columns: 1fr 1fr;
+        align-items: baseline;
+        row-gap: ${theme.space[3]}px;
+      `}
+    >
+      <H4
+        css={`
+          grid-column: 1/3;
+        `}
+      >
+        Global Settings
+      </H4>
+
+      <OptionLabel htmlFor={maxSessionTTLId}>Max Session TTL</OptionLabel>
+      <Input
+        id={maxSessionTTLId}
+        value={value.maxSessionTTL}
+        disabled={isProcessing}
+        onChange={e => onChange({ ...value, maxSessionTTL: e.target.value })}
+      />
+
+      <OptionLabel htmlFor={clientIdleTimeoutId}>
+        Client Idle Timeout
+      </OptionLabel>
+      <Input
+        id={clientIdleTimeoutId}
+        value={value.clientIdleTimeout}
+        disabled={isProcessing}
+        onChange={e =>
+          onChange({ ...value, clientIdleTimeout: e.target.value })
+        }
+      />
+
+      <div>Disconnect When Certificate Expires</div>
+      <BoolRadioGroup
+        name="disconnect-expired-cert"
+        value={value.disconnectExpiredCert}
+        onChange={d => onChange({ ...value, disconnectExpiredCert: d })}
+      />
+
+      <OptionLabel htmlFor={requireMFATypeId}>Require Session MFA</OptionLabel>
+      <Select
+        inputId={requireMFATypeId}
+        isDisabled={isProcessing}
+        options={requireMFATypeOptions}
+        value={value.requireMFAType}
+        onChange={t => onChange?.({ ...value, requireMFAType: t })}
+      />
+
+      <H4
+        css={`
+          grid-column: 1/3;
+          border-top: ${theme.borders[1]}
+            ${theme.colors.interactive.tonal.neutral[0]};
+          padding-top: ${theme.space[3]}px;
+        `}
+      >
+        SSH
+      </H4>
+
+      <OptionLabel htmlFor={createHostUserModeId}>
+        Create Host User Mode
+      </OptionLabel>
+      <Select
+        inputId={createHostUserModeId}
+        isDisabled={isProcessing}
+        options={createHostUserModeOptions}
+        value={value.createHostUserMode}
+        onChange={m => onChange?.({ ...value, createHostUserMode: m })}
+      />
+
+      <H4
+        css={`
+          grid-column: 1/3;
+          border-top: ${theme.borders[1]}
+            ${theme.colors.interactive.tonal.neutral[0]};
+          padding-top: ${theme.space[3]}px;
+        `}
+      >
+        Database
+      </H4>
+
+      <div>Create Database User</div>
+      <BoolRadioGroup
+        name="create-db-user"
+        value={value.createDBUser}
+        onChange={c => onChange({ ...value, createDBUser: c })}
+      />
+
+      {/* TODO(bl-nero): a bug in YAML unmarshalling backend breaks this option. Fix it. */}
+      {/* <OptionLabel htmlFor={createDBUserModeId}>
+        Create Database User Mode
+      </OptionLabel>
+      <Select
+        inputId={createDBUserModeId}
+        isDisabled={isProcessing}
+        options={createDBUserModeOptions}
+        value={value.createDBUserMode}
+        onChange={m => onChange?.({ ...value, createDBUserMode: m })}
+      /> */}
```

Comment:
Sometime back, I was suggested not to merge such commented code to master. Leaving a TODO note is fine and then you can bring the code back once you have the solution.